internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -157,75 +153,25 @@ func installElasticAgentInCluster() error {
 	return nil
 }
 
-// downloadElasticAgentManagedYAML will download a url from a path and return the response body.
-func downloadElasticAgentManagedYAML(url string) ([]byte, error) {
```

Comment:
To be honest I'm on the fence with this change as it isn't inlined with idea we had before - have a single place where we store k8s definitions, so that the correct manifest is picked up without any updates in elastic-package's codebase.

Maybe we should discuss possible options here?